shared/Utils.ts

```diff
@@ -28,7 +28,11 @@ export interface CommunicationDetails {
 }
 
 export const isMosipVC = (issuer: string) => {
-  return issuer === Issuers.Mosip || issuer === Issuers.MosipOtp;
+  return (
+    issuer === Issuers.Mosip ||
+    issuer === Issuers.MosipOtp ||
+    issuer === Issuers.MosipCertify
```

Comment:
We don't need to add this.
We are supported only 4 issuers
1. MosipOtp
2. Mosip
3. StayProtected
4. Mock
We need this hardcoding only for Mosip VC as Mosip VC requires activation which we don't get to know from any property.

Reply:
Removed certify condition